src/h3/api/_api_template.py

```diff
@@ -120,10 +160,12 @@ def h3_to_parent(h, res=None):
         return p
 
     def h3_distance(h1, h2):
-        """ Compute the hex-distance between two hexagons
+        """ Compute the H3-distance between two cells
```

Comment:
"H3-distance" -> "distance in number of grid cells"?

Reply:
I've updated the description. Let me know what you think!